### PR TITLE
spacing problem between dropdowns on dropdown docs

### PR DESCRIPTION
**Pull request by mznet**
image

I've found that a spacing problem between dropdowns on docs. This problem is caused by left margin even though line breaks. To fix problem, I removed left margin between dropdowns on xs size and applied the margin on over sm size.

You can check the issue on [Sizing](https://v5.getbootstrap.com/docs/5.0/components/dropdowns/#sizing) section of dropdown page. Check it on xs size screen. And this issue is happened to v4 as well as v5.

Preview: https://deploy-preview-31841--twbs-bootstrap.netlify.app/docs/5.0/components/dropdowns/#sizing

### CODE REVIEW

File: site/content/docs/5.0/components/dropdowns.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ Button dropdowns work with buttons of all sizes, including default and split dro
         <li><a class="dropdown-item" href="#">Separated link</a></li>
       </ul>
     </div><!-- /btn-group -->
-    <div class="btn-group ml-2">
+    <div class="btn-group ml-sm-2">
```
Comment (mdo) [inline site/content/docs/5.0/components/dropdowns.md:263]: This now no longer matches L290, where we still use `ml-2`. I'd suggest deleting both `ml-2` instances instead.

**Review by mdo — DISMISSED** [review]

File: site/content/docs/5.0/components/dropdowns.md (hunk 1 above)
Comment (mznet) [reply site/content/docs/5.0/components/dropdowns.md]: @mdo set `L263` and `L290` the same with deleting both `ml-2`.
I'd adjusted margin of `> .btn-toolbar > .btn-group,` in the previous commit and I realized that it affected button group docs. So, I additionally adjusted margin of `> .btn-toolbar > .input-group` to set proper margin in button group docs

**Review by MartijnCuppens — APPROVED** [review]
LGTM, thanks @mznet!

**Comment by mdo** [comment]
Replacing this with #31861—while reviewing, I realized we could split those examples and avoid unnecessary changes to our components CSS (which negatively impact all many other `btn-group`s in other snippets).

**Pull request closed without merging**